bin/start

```diff
@@ -82,4 +87,12 @@ if [ ! -z "$RDS_HOSTNAME" ]; then
     export MB_DB_PORT=$RDS_PORT
 fi
 
-exec java -Dfile.encoding=UTF-8 $JAVA_OPTS -jar ./target/uberjar/metabase.jar
+# Setup Java Options
+JAVA_OPTS="${JAVA_OPTS} -Dlogfile.path=target/log -XX:+CMSClassUnloadingEnabled -XX:+UseConcMarkSweepGC -server"
+
+if [ ! -z "$JAVA_TIMEZONE" ]; then
+  JAVA_OPTS="${JAVA_OPTS} -Duser.timezone=${JAVA_TIMEZONE}"
+fi
+
+
+exec java -Dfile.encoding=UTF-8 $JAVA_OPTS -jar ./metabase.jar
```

Comment:
This is probably going to break pushing to Heroku to deploy, since it basically just runs `bin/build` then `bin/start`.

 If we do this I think we should have ./bin/build move the jar from `./target/uberjar/metabase.jar` to `./metabase.jar` (and we'll need to update the E2E test runner)